phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/PhoenixIndexImportDirectReducer.java

```diff
@@ -49,8 +50,27 @@
     private static final Logger LOGGER =
             LoggerFactory.getLogger(PhoenixIndexImportDirectReducer.class);
 
+    private static Integer regionCountForTesting = null;
+
+    public static void setRegionCountForTesting(Integer regionCount) {
```

Comment:
why Integer instead of int?

Reply:
So that I can pass null. It does null check. With Integer you know it is not set. With int, you dont

Reply:
Okay. It would be nice to go back to primitive DT with a non-set value as -1 or any non-negative number of your choice.